app/pages/home-for-user/index.jsx

```diff
@@ -120,10 +125,36 @@ const HomePageForUser = React.createClass({
     });
   },
 
+  recentSubjects(ribbonData) {
+    this.setState({
+      newData: this.fetchRecentSubjects(ribbonData),
+    });
+  },
+
   findProjectLink(project) {
     return `/projects/${project.slug}`;
   },
 
+  fetchRecentSubjects(ribbonData) {
+    const newestSubjects = []
+    ribbonData.map((project) => {
+      if (project['recentSubjectSet'] !== undefined){
+        newestSubjects.push(project)
+      }
+    });
+    const recentSets = newestSubjects.sort((a, b) => {
```

Comment:
This sort can be done with the API, so doing this manually here isn't necessary. Also, it'll only be sorting the first page of subject sets returned.